Lh625 default configuration settings

MikeThomsen opened the pull request.

MikeThomsen committed:
[#553 state:resolved] Moved the spanish translation of DePoClean from MT::L10N to DePoClean::L10N.

MikeThomsen committed:
[#343 state:resolved] Added support for autodirifcation of assets and two config directives to control the behavior.

MikeThomsen committed:
[#522 state:resolved] Changed copyright notice in build/minifier.pl and a few other files; new builds should now automatically pick up Melody copyright notice.

MikeThomsen committed:
[#299 state:resolved] Integrated the widget set loop functionality into Melody.

byrnereese committed:
[#635 state:resolved] Dramatically improved theme dashboard usability. Now supports deep-linking, and hitting the browser back button takes you to the previously selected tab as opposed to the previous screen.

MikeThomsen committed:
[#627 state:resolved] Changed link on debug mode docs from mt.org to melody wiki.

MikeThomsen committed:
[#625 state:resolved] Changed the I18N/default.pm values for LearningNewsURL, ExportEncoding and LogExportEncoding to make them the same as MT::Core.

jayallen commented:
Erm, are you sure you were trying to fix [Bug 627](https://openmelody.lighthouseapp.com/projects/26604-melody/tickets/627)

jayallen commented:
It looks like you mean ticket 625...

MikeThomsen committed:
[#619 state:resolved] Fixed a bug in MT::CMS::System which prevented the default timezone from being updated.

jayallen committed:
[#654 state:resolved] Synched Melody with changes from Movable Type 4.35. See http://www.movabletype.org/2010/12/movable_type_504_and_435_security_update.html

byrnereese committed:
[#550 state:resolved] Fixed the "Design Your Blog" link to point to the "Apply Theme" screen on theme dashboard.

Commit:
[#655 state:resolved] Upgrade finished page now shows correct version number (`MT->version_id` instead of `MT->version_number`)

Commit:
[#657 state:resolved] Upgrader app now correctly checks for perl version 5.8.8 instead of perl version 5.6.1

byrnereese committed:
[#653 state:resolved] Added link to help documentation for how to use revision histories.

byrnereese committed:
[#651 state:resolved] Note about Image::Magick not being installed now only appears to system administrators.

Upon investigation of this ticket, it was learned that the message that appears when the support directory is not writable is already restricted to administrators.

Commit:
[#656 state:resolved] [#658 state:resolved] Fixed use of version number (i.e. API version or corresponding MT version) for basis of Melody upgrades.  Now properly uses product version (i.e. Melody's version).  Also fixed two translated strings and several translations to properly report the product name.

Commit:
[#659 state:resolved] Converted all docs links from use of http://github.com/openmelody/melody/wiki to http://openmelody.org/docs/.  These are currently aliased but won't be in the future.

jayallen committed:
Merge branch 'byrnereese-lh-651-653-small-changes' into release-1.0b2

* byrnereese-lh-651-653-small-changes:
  [#659 state:resolved] Converted all docs links from use of http://github.com/openmelody/melody/wiki to http://openmelody.org/docs/.  These are currently aliased but won't be in the future.
  [#651 state:resolved] Note about Image::Magick not being installed now only appears to system administrators.
  [#653 state:resolved] Added link to help documentation for how to use revision histories.
